### ISSUE TITLE
[BUG] Parameters of function call get into the next function calls.

### ISSUE THREAD

**Issue by Danmyrer** [Azure/azure-functions-sql-extension#390]
###### Please provide the following:

- Timestamp: 2022-09-25T12:43:18Z
- Function App name: smartFME-API-Functions
- Function name(s) (as appropriate): dFunc_manageDomPlattformen, dFunc_manageDomArtPersonen
- Core Tools version: [4.0.4785](https://github.com/Azure/azure-functions-core-tools/releases/tag/4.0.4785)
- Extension Bundle: Microsoft.Azure.Functions.ExtensionBundle.Preview  4.*, 5.0.0

###### Provide the steps required to reproduce the problem:

1. Call function dFunc* (any) with any action (i.e "action": "update")
2. Call the same function again with different action (i.e "action": "delete")

In my Case:
```json
{
    "action": "update",
    "data":
    {
        "DOM_Plattform_ID": "4",
        "Name": "Macintosh"
    }
}
```
```json
{
    "action": "delete",
    "data":
    {
        "DOM_Plattform_ID": "12"
    }
}
```

###### Provide a description of the expected behavior.

Second function call gets successfully executed.

###### Provide a description of the actual behavior observed.

Function throws an error:
```
2022-09-25T12:43:18Z   [Information]   Sending event Create
2022-09-25T12:43:18Z   [Information]   Sending event AddAsync
2022-09-25T12:43:18Z   [Information]   Sending event FlushAsync
2022-09-25T12:43:18Z   [Information]   Sending event TableInfoCacheHit
2022-09-25T12:43:18Z   [Information]   Sending event UpsertStart
2022-09-25T12:43:18Z   [Information]   Sending exception event: Invalid column name 'Inaktiv'.
Invalid column name 'Inaktiv'.
2022-09-25T12:43:18Z   [Information]   Sending exception event: Invalid column name 'Inaktiv'.
Invalid column name 'Inaktiv'.
2022-09-25T12:43:18Z   [Error]   Executed 'Functions.dFunc_manageDomPlattformen' (Failed, Id=85862a6c-8791-47ba-b295-b65b05985079, Duration=62ms)
```
The problem with this is, that neither the second function call or the responsible code included the column 'Inaktiv'. The first function call however, did include the column (in my case delete simply adds 'Inaktiv = True' to the specified row).    

###### Provide a description of any known workarounds.

Restart Core Tools

###### Provide the requirements.txt file to help us find out module related issues.

```
azure-functions==1.11.3b1
argon2-cffi
```

###### Provide any related information

<details>
<summary>Source</summary>

##### dFunc_manageDomPlattformen

<details>
<summary>function.json</summary>

```json
{
  "scriptFile": "__init__.py",
  "bindings": [
    {
      "authLevel": "function",
      "type": "httpTrigger",
      "direction": "in",
      "name": "REQ",
      "methods": [
        "get",
        "post"
      ]
    },
    {
      "type": "http",
      "direction": "out",
      "name": "$return"
    },
    {
      "name": "DOMPlattformenIN",
      "type": "sql",
      "direction": "in",
      "commandText": "SELECT * FROM DOM.DOM_Plattformen",
      "commandType": "Text",
      "connectionStringSetting": "SqlConnectionString"
    },
    {
      "name": "DOMPlattformenOUT",
      "type": "sql",
      "direction": "out",
      "commandText": "DOM.DOM_Plattformen",
      "connectionStringSetting": "SqlConnectionString"
    }
  ]
}
```

</details>

<details>
<summary>__init.py__</summary>

```python
import logging
import json

from Tools.httpRequest import getParameter
from Tools.sql import mapData
from Tools.crud import crud

import azure.functions as func


def main(REQ: func.HttpRequest, DOMPlattformenIN: func.SqlRowList, DOMPlattformenOUT: func.Out[func.SqlRow]) -> func.HttpResponse:
    logging.info('Python HTTP trigger function processed a request.')

    ID_STRING = "DOM_Plattform_ID"

    ACTION = getParameter(REQ, "action")
    DATA = getParameter(REQ, "data")

    if ACTION is None:
        return func.HttpResponse("action must not be null", status_code=400)
    
    DOMPlattformen = mapData(DOMPlattformenIN)

    logging.info(json.dumps(DATA))
    
    return crud(ACTION, DATA, DOMPlattformen, DOMPlattformenOUT, ID_STRING)
```

</details>

<details>
<summary>crud.py partially</summary>

```python
import json
import logging
import traceback
import azure.functions as func
import Tools.crudActions as crudActions

def crud(action, data, tableIn, tableOut, id_string):

    elif action == "update":
        try:
            crudActions.updateRow(data, tableIn, tableOut, id_string)
            return func.HttpResponse("Row wurde erfolgreich aktualisiert!")
        except Exception as ex:
            if hasattr(ex, "httpResponse"):
                return ex.httpResponse
            logging.info(traceback.print_exc())
            return func.HttpResponse("Something went wrong...", status_code=500)
    
    elif action == "delete":
        try:
            crudActions.deleteRow(data, tableIn, tableOut, id_string)
            return func.HttpResponse("Row erfolgreich gelöscht!")
        except Exception as ex:
            if hasattr(ex, "httpResponse"):
                return ex.httpResponse
            logging.info(traceback.print_exc())
            return func.HttpResponse("Something went wrong...", status_code=500)

    return func.HttpResponse("action =! 'create', 'read', 'update', 'delete'", status_code=400)
```

</details>

<details>
<summary>crudActions.py (partially)</summary>

```python
import logging
import azure.functions as func
import Tools.exception as e

import Tools.sql as helper


def updateRow(data, tableIn, tableOut, pkName):
    logging.info("updateRow...")

    # Prüfung, ob Personen_ID in DB existiert
    if not pkName in data:
        logging.info("updateRow >> False | Keine ID angegeben")
        raise e.MissingDataError()
    if not helper.checkForId(tableIn, pkName, data[pkName]):
        logging.info("updateRow >> False | ID nicht gefunden!")
        raise e.IdNotFoundError()

    row = func.SqlRow.from_dict(data)
    try:
        tableOut.set(row)
        logging.info("updateRow >> True")
        return
    except:
        logging.info("updateRow >> False")
        raise e.SqlError()

def deleteRow(data, tableIn, tableOut: func.Out[func.SqlRow], pkName):
    logging.info("deleteRow...")

    # Prüfung, ob ID in DB existiert
    if not pkName in data:
        logging.info("deleteRow >> False | Keine ID angegeben")
        raise e.MissingDataError()
    elif len(data.keys()) > 1:
        logging.info(f"deleteRow >> False | deleteRow kann nur '{pkName}' akzeptieren")
        raise e.TooMuchDataError()
    if not helper.checkForId(tableIn, pkName, data[pkName]):
        logging.info("deleteRow >> False | ID nicht gefunden!")
        raise e.IdNotFoundError()

    # Delete User "löscht" keine Reihen, sondern markiert sie als Inaktiv, dadurch werden sie von den Anderen APIs ignoriert.
    data["Inaktiv"] = True
    # Delete User doesn't "delete" Rows, it just marks them as 'Inaktiv'

    row = func.SqlRow.from_dict(data)
    try:
        logging.info(f"Row: {row.data}")
        tableOut.set(row)
        logging.info(tableOut.get().data)
        logging.info("deleteRow >> True")
        return True
    except:
        logging.info("deleteRow >> False")
        raise e.SqlError()
```

</details>

<details>
<summary>Full Logs</summary>

```txt
2022-09-25T12:42:50Z   [Information]   Python HTTP trigger function processed a request.
2022-09-25T12:42:50Z   [Information]   Retrieved the following Parameter: "action": "delete"
2022-09-25T12:42:50Z   [Information]   Retrieved the following Parameter: "data": "{'DOM_Plattform_ID': '15'}"
2022-09-25T12:42:50Z   [Information]   {"DOM_Plattform_ID": "15"}
2022-09-25T12:42:50Z   [Information]   deleteRow...
2022-09-25T12:42:50Z   [Information]   checkForId...
2022-09-25T12:42:50Z   [Information]   checkForId >> True
2022-09-25T12:42:50Z   [Information]   deleteRow >> True
2022-09-25T12:42:51Z   [Information]   Sending event Create
2022-09-25T12:42:51Z   [Information]   Sending event AddAsync
2022-09-25T12:42:51Z   [Information]   Sending event FlushAsync
2022-09-25T12:42:51Z   [Information]   Sending event TableInfoCacheMiss
2022-09-25T12:42:51Z   [Information]   Sending event GetTableInfoStart
2022-09-25T12:42:51Z   [Information]   Sending event GetCaseSensitivity
2022-09-25T12:42:51Z   [Information]   Sending event GetColumnDefinitions
2022-09-25T12:42:51Z   [Information]   Sending event GetPrimaryKeys
2022-09-25T12:42:51Z   [Information]   Sending event GetTableInfoEnd
2022-09-25T12:42:51Z   [Information]   DB and Table: smartFME-SQL.DOM.DOM_Plattformen. Primary keys: []. SQL Column and Definitions:  [dom_plattform_id int,name varchar(255),inaktiv bit]
2022-09-25T12:42:51Z   [Information]   Sending event UpsertStart
2022-09-25T12:42:51Z   [Information]   Sending event UpsertEnd
2022-09-25T12:42:51Z   [Information]   Upserted 1 row(s) into database: smartFME-SQL and table: DOM.DOM_Plattformen.
2022-09-25T12:42:51Z   [Information]   Executed 'Functions.dFunc_manageDomPlattformen' (Succeeded, Id=8717f9c4-4145-41dd-9b8b-22012d6d0292, Duration=536ms)
2022-09-25T12:43:18Z   [Information]   Executing 'Functions.dFunc_manageDomPlattformen' (Reason='This function was programmatically called via the host APIs.', Id=85862a6c-8791-47ba-b295-b65b05985079)
2022-09-25T12:43:18Z   [Information]   Sending event Create
2022-09-25T12:43:18Z   [Information]   Sending event Convert
2022-09-25T12:43:18Z   [Information]   13 row(s) queried from database: smartFME-SQL using Command: SELECT * FROM DOM.DOM_Plattformen
2022-09-25T12:43:18Z   [Verbose]   Sending invocation id:85862a6c-8791-47ba-b295-b65b05985079
2022-09-25T12:43:18Z   [Verbose]   Posting invocation id:85862a6c-8791-47ba-b295-b65b05985079 on workerId:96b523d8-35a8-4b9e-a51c-8b1a2f56b034
2022-09-25T12:43:18Z   [Information]   Python HTTP trigger function processed a request.
2022-09-25T12:43:18Z   [Information]   Retrieved the following Parameter: "action": "update"
2022-09-25T12:43:18Z   [Information]   Retrieved the following Parameter: "data": "{'DOM_Plattform_ID': '4', 'Name': 'Macintosh'}"
2022-09-25T12:43:18Z   [Information]   {"DOM_Plattform_ID": "4", "Name": "Macintosh"}
2022-09-25T12:43:18Z   [Information]   updateRow...
2022-09-25T12:43:18Z   [Information]   checkForId...
2022-09-25T12:43:18Z   [Information]   checkForId >> True
2022-09-25T12:43:18Z   [Information]   updateRow >> True
2022-09-25T12:43:18Z   [Information]   Sending event Create
2022-09-25T12:43:18Z   [Information]   Sending event AddAsync
2022-09-25T12:43:18Z   [Information]   Sending event FlushAsync
2022-09-25T12:43:18Z   [Information]   Sending event TableInfoCacheHit
2022-09-25T12:43:18Z   [Information]   Sending event UpsertStart
2022-09-25T12:43:18Z   [Information]   Sending exception event: Invalid column name 'Inaktiv'.
Invalid column name 'Inaktiv'.
2022-09-25T12:43:18Z   [Information]   Sending exception event: Invalid column name 'Inaktiv'.
Invalid column name 'Inaktiv'.
2022-09-25T12:43:18Z   [Error]   Executed 'Functions.dFunc_manageDomPlattformen' (Failed, Id=85862a6c-8791-47ba-b295-b65b05985079, Duration=62ms)
```

</details>

</details>

**Comment by Danmyrer** [comment]
Sure! (@Charles-Gagnon sorry for the delay, I was on vacation.)

 <details>
<summary>Debug Logs</summary>

```log

[2022-10-21T17:28:50.162Z] Executing HTTP request: {
[2022-10-21T17:28:50.162Z]   requestId: "90c48cad-944b-408c-81fd-a4f9d84756b9",
[2022-10-21T17:28:50.162Z]   method: "POST",
[2022-10-21T17:28:50.162Z]   userAgent: "ms-rest-js/2.6.0 Node/v16.14.2 OS/(x64-Linux-5.19.14-200.fc36.x86_64) vscode-azurefunctions/1.8.1",
[2022-10-21T17:28:50.162Z]   uri: "/api/dFunc_manageDomPlattformen"
[2022-10-21T17:28:50.162Z] }
[2022-10-21T17:28:50.173Z] Request successfully matched the route with name 'dFunc_manageDomPlattformen' and template 'api/dFunc_manageDomPlattformen'
[2022-10-21T17:28:50.280Z] Executing 'Functions.dFunc_manageDomPlattformen' (Reason='This function was programmatically called via the host APIs.', Id=f4f11487-2e54-4c66-bc59-c747592cdb48)
[2022-10-21T17:28:50.332Z] Sending event Create
[2022-10-21T17:28:50.343Z] TID:15 BEGIN ConvertAsync (Json)
[2022-10-21T17:28:50.882Z] Sending event Convert
[2022-10-21T17:28:50.960Z] 5 row(s) queried from database: smartFME-SQL using Command: SELECT * FROM DOM.DOM_Plattformen
[2022-10-21T17:28:50.979Z] TID:21 END ConvertAsync (Json) Duration=635ms
[2022-10-21T17:28:51.039Z] Received FunctionInvocationRequest, request ID: 6471ad3a-0f42-404c-ab51-847f32220f70, function ID: 8e4cd30c-91c5-4218-9207-049dd69d73f2, function name: dFunc_manageDomPlattformen, invocation ID: f4f11487-2e54-4c66-bc59-c747592cdb48, function type: sync, sync threadpool max workers: 1000
[2022-10-21T17:28:51.052Z] Python HTTP trigger function processed a request.
[2022-10-21T17:28:51.053Z] Retrieved the following Parameter: "action": "create"
[2022-10-21T17:28:51.054Z] Retrieved the following Parameter: "data": "{'Name': 'Something'}"
[2022-10-21T17:28:51.054Z] {"Name": "Something"}
[2022-10-21T17:28:51.054Z] createRow...
[2022-10-21T17:28:51.054Z] getFreeId...
[2022-10-21T17:28:51.054Z] getFreeId >> 5
[2022-10-21T17:28:51.054Z] createRow >> True
[2022-10-21T17:28:51.072Z] InvocationResponse received for invocation id: 'f4f11487-2e54-4c66-bc59-c747592cdb48'
[2022-10-21T17:28:51.137Z] Sending event Create
[2022-10-21T17:28:51.139Z] Sending event AddAsync
[2022-10-21T17:28:51.147Z] Sending event FlushAsync
[2022-10-21T17:28:51.159Z] TID:15 BEGIN UpsertRowsAsync
[2022-10-21T17:28:51.170Z] Sending event TableInfoCacheMiss
[2022-10-21T17:28:51.185Z] Sending event GetTableInfoStart
[2022-10-21T17:28:51.185Z] TID:15 BEGIN RetrieveTableInformationAsync
[2022-10-21T17:28:51.261Z] TID:15 BEGIN GetCaseSensitivity Query="
[2022-10-21T17:28:51.261Z]                     SELECT
[2022-10-21T17:28:51.261Z]                         DATABASEPROPERTYEX('smartFME-SQL', 'Collation') AS Collation;"
[2022-10-21T17:28:51.313Z] Sending event GetCaseSensitivity
[2022-10-21T17:28:51.313Z] TID:4 END GetCaseSensitivity Duration=50ms
[2022-10-21T17:28:51.313Z] TID:4 BEGIN GetColumnDefinitions Query="
[2022-10-21T17:28:51.313Z]                     select
[2022-10-21T17:28:51.313Z]                          COLUMN_NAME, DATA_TYPE +
[2022-10-21T17:28:51.313Z]                                  case
[2022-10-21T17:28:51.314Z]                                          when CHARACTER_MAXIMUM_LENGTH = -1 then '(max)'
[2022-10-21T17:28:51.314Z]                                          when CHARACTER_MAXIMUM_LENGTH <> -1 then '(' + cast(CHARACTER_MAXIMUM_LENGTH as varchar(4)) + ')'
[2022-10-21T17:28:51.314Z]                                 when DATETIME_PRECISION is not null and DATA_TYPE not in ('datetime', 'date', 'smalldatetime') then '(' + cast(DATETIME_PRECISION as varchar(1)) + ')'
[2022-10-21T17:28:51.314Z]                                          when DATA_TYPE in ('decimal', 'numeric') then '(' + cast(NUMERIC_PRECISION as varchar(9)) + ',' + + cast(NUMERIC_SCALE as varchar(9)) + ')'
[2022-10-21T17:28:51.314Z]                                          else ''
[2022-10-21T17:28:51.314Z]                                  end as COLUMN_DEFINITION
[2022-10-21T17:28:51.314Z]                     from
[2022-10-21T17:28:51.314Z]                          INFORMATION_SCHEMA.COLUMNS c
[2022-10-21T17:28:51.314Z]                     where
[2022-10-21T17:28:51.314Z]                          c.TABLE_NAME = 'DOM_Plattformen'
[2022-10-21T17:28:51.314Z]                     and
[2022-10-21T17:28:51.314Z]                         c.TABLE_SCHEMA = 'DOM'"
[2022-10-21T17:28:51.336Z] Sending event GetColumnDefinitions
[2022-10-21T17:28:51.336Z] TID:15 END GetColumnDefinitions Duration=22ms
[2022-10-21T17:28:51.337Z] TID:15 BEGIN GetPrimaryKeys Query="
[2022-10-21T17:28:51.337Z]                     SELECT
[2022-10-21T17:28:51.337Z]                         COLUMN_NAME, c.is_identity
[2022-10-21T17:28:51.337Z]                     FROM
[2022-10-21T17:28:51.337Z]                         INFORMATION_SCHEMA.TABLE_CONSTRAINTS tc
[2022-10-21T17:28:51.337Z]                     INNER JOIN
[2022-10-21T17:28:51.337Z]                         INFORMATION_SCHEMA.CONSTRAINT_COLUMN_USAGE ccu ON ccu.CONSTRAINT_NAME = tc.CONSTRAINT_NAME AND ccu.TABLE_NAME = tc.TABLE_NAME
[2022-10-21T17:28:51.337Z]                     INNER JOIN
[2022-10-21T17:28:51.337Z]                         sys.columns c ON c.object_id = OBJECT_ID('[DOM].[DOM_Plattformen]') AND c.name = ccu.COLUMN_NAME
[2022-10-21T17:28:51.337Z]                     WHERE
[2022-10-21T17:28:51.337Z]                         tc.CONSTRAINT_TYPE = 'PRIMARY KEY'
[2022-10-21T17:28:51.337Z]                     and
[2022-10-21T17:28:51.337Z]                         tc.TABLE_NAME = 'DOM_Plattformen'
[2022-10-21T17:28:51.337Z]                     and
[2022-10-21T17:28:51.337Z]                         tc.TABLE_SCHEMA = 'DOM'"
[2022-10-21T17:28:51.360Z] Sending event GetPrimaryKeys
[2022-10-21T17:28:51.361Z] TID:4 END GetPrimaryKeys Duration=24ms
[2022-10-21T17:28:51.371Z] Sending event GetTableInfoEnd
[2022-10-21T17:28:51.377Z] TID:4 END RetrieveTableInformationAsync Duration=107ms DB and Table: smartFME-SQL.DOM.DOM_Plattformen. Primary keys: []. SQL Column and Definitions:  [[dom_plattform_id, int],[name, varchar(255)],[inaktiv, bit]]
[2022-10-21T17:28:51.390Z] Sending event UpsertStart
[2022-10-21T17:28:51.390Z] TID:4 BEGIN UpsertRowsTransaction
[2022-10-21T17:28:51.523Z] Sending event UpsertEnd
[2022-10-21T17:28:51.523Z] TID:32 END UpsertRowsTransaction Duration=133ms Upserted 1 row(s) into database: smartFME-SQL and table: DOM.DOM_Plattformen.
[2022-10-21T17:28:51.524Z] TID:32 END UpsertRowsAsync Duration=364ms
[2022-10-21T17:28:51.543Z] Executed 'Functions.dFunc_manageDomPlattformen' (Succeeded, Id=f4f11487-2e54-4c66-bc59-c747592cdb48, Duration=1299ms)
[2022-10-21T17:28:51.569Z] Executed HTTP request: {
[2022-10-21T17:28:51.569Z]   requestId: "90c48cad-944b-408c-81fd-a4f9d84756b9",
[2022-10-21T17:28:51.570Z]   identities: "(WebJobsAuthLevel:Admin)",
[2022-10-21T17:28:51.570Z]   status: "200",
[2022-10-21T17:28:51.570Z]   duration: "1407"
[2022-10-21T17:28:51.570Z] }
[2022-10-21T17:29:09.962Z] Executing HTTP request: {
[2022-10-21T17:29:09.962Z]   requestId: "942e31e8-b411-4111-b5de-cfde0c4f02c3",
[2022-10-21T17:29:09.962Z]   method: "GET",
[2022-10-21T17:29:09.962Z]   userAgent: "ms-rest-js/2.6.0 Node/v16.14.2 OS/(x64-Linux-5.19.14-200.fc36.x86_64) vscode-azurefunctions/1.8.1",
[2022-10-21T17:29:09.962Z]   uri: "/"
[2022-10-21T17:29:09.962Z] }
[2022-10-21T17:29:09.964Z] Executed HTTP request: {
[2022-10-21T17:29:09.965Z]   requestId: "942e31e8-b411-4111-b5de-cfde0c4f02c3",
[2022-10-21T17:29:09.965Z]   identities: "",
[2022-10-21T17:29:09.965Z]   status: "200",
[2022-10-21T17:29:09.965Z]   duration: "2"
[2022-10-21T17:29:09.965Z] }
[2022-10-21T17:29:09.994Z] Executing HTTP request: {
[2022-10-21T17:29:09.994Z]   requestId: "22ccac80-e035-427c-8525-955d84a996af",
[2022-10-21T17:29:09.994Z]   method: "POST",
[2022-10-21T17:29:09.994Z]   userAgent: "ms-rest-js/2.6.0 Node/v16.14.2 OS/(x64-Linux-5.19.14-200.fc36.x86_64) vscode-azurefunctions/1.8.1",
[2022-10-21T17:29:09.994Z]   uri: "/api/dFunc_manageDomPlattformen"
[2022-10-21T17:29:09.994Z] }
[2022-10-21T17:29:09.994Z] Request successfully matched the route with name 'dFunc_manageDomPlattformen' and template 'api/dFunc_manageDomPlattformen'
[2022-10-21T17:29:10.001Z] Executing 'Functions.dFunc_manageDomPlattformen' (Reason='This function was programmatically called via the host APIs.', Id=1c96e9f7-ad28-4b61-9006-1440a398c6b2)
[2022-10-21T17:29:10.004Z] Sending event Create
[2022-10-21T17:29:10.004Z] TID:4 BEGIN ConvertAsync (Json)
[2022-10-21T17:29:10.004Z] Sending event Convert
[2022-10-21T17:29:10.028Z] 6 row(s) queried from database: smartFME-SQL using Command: SELECT * FROM DOM.DOM_Plattformen
[2022-10-21T17:29:10.028Z] TID:4 END ConvertAsync (Json) Duration=23ms
[2022-10-21T17:29:10.036Z] Received FunctionInvocationRequest, request ID: 6471ad3a-0f42-404c-ab51-847f32220f70, function ID: 8e4cd30c-91c5-4218-9207-049dd69d73f2, function name: dFunc_manageDomPlattformen, invocation ID: 1c96e9f7-ad28-4b61-9006-1440a398c6b2, function type: sync, sync threadpool max workers: 1000
[2022-10-21T17:29:10.040Z] Python HTTP trigger function processed a request.
[2022-10-21T17:29:10.040Z] Retrieved the following Parameter: "action": "delete"
[2022-10-21T17:29:10.040Z] Retrieved the following Parameter: "data": "{'DOM_Plattform_ID': '5'}"
[2022-10-21T17:29:10.041Z] {"DOM_Plattform_ID": "5"}
[2022-10-21T17:29:10.041Z] deleteRow...
[2022-10-21T17:29:10.042Z] checkForId...
[2022-10-21T17:29:10.042Z] checkForId >> True
[2022-10-21T17:29:10.042Z] Row: {'DOM_Plattform_ID': '5', 'Inaktiv': True}
[2022-10-21T17:29:10.043Z] {'DOM_Plattform_ID': '5', 'Inaktiv': True}
[2022-10-21T17:29:10.043Z] deleteRow >> True
[2022-10-21T17:29:10.044Z] InvocationResponse received for invocation id: '1c96e9f7-ad28-4b61-9006-1440a398c6b2'
[2022-10-21T17:29:10.046Z] Sending event Create
[2022-10-21T17:29:10.046Z] Sending event AddAsync
[2022-10-21T17:29:10.047Z] Sending event FlushAsync
[2022-10-21T17:29:10.047Z] TID:10 BEGIN UpsertRowsAsync
[2022-10-21T17:29:10.053Z] Sending event TableInfoCacheHit
[2022-10-21T17:29:10.054Z] Sending event UpsertStart
[2022-10-21T17:29:10.054Z] TID:10 BEGIN UpsertRowsTransaction
[2022-10-21T17:29:10.104Z] Sending exception event: Invalid column name 'Name'.
[2022-10-21T17:29:10.104Z] Invalid column name 'Name'.
[2022-10-21T17:29:10.128Z] Sending exception event: Invalid column name 'Name'.
[2022-10-21T17:29:10.129Z] Invalid column name 'Name'.
[2022-10-21T17:29:10.143Z] Executed 'Functions.dFunc_manageDomPlattformen' (Failed, Id=1c96e9f7-ad28-4b61-9006-1440a398c6b2, Duration=140ms)
[2022-10-21T17:29:10.143Z] System.Private.CoreLib: Exception while executing function: Functions.dFunc_manageDomPlattformen. Microsoft.Azure.WebJobs.Host: Error while handling parameter _binder after function returned:. Core Microsoft SqlClient Data Provider: Invalid column name 'Name'.
[2022-10-21T17:29:10.143Z] Invalid column name 'Name'.
[2022-10-21T17:29:10.146Z] Executed HTTP request: {
[2022-10-21T17:29:10.146Z]   requestId: "22ccac80-e035-427c-8525-955d84a996af",
[2022-10-21T17:29:10.146Z]   identities: "(WebJobsAuthLevel:Admin)",
[2022-10-21T17:29:10.146Z]   status: "500",
[2022-10-21T17:29:10.146Z]   duration: "152"
[2022-10-21T17:29:10.146Z] }

```

**Comment by lucyzhang929** [comment]
@Danmyrer A new extension bundle version has been released (v4.6.0). Could you test your scenario again and let us know if you are still seeing this issue? Thank you!

**Comment by lucyzhang929** [comment]
Closing this since it should be fixed. @Danmyrer feel free to reopen if you are still seeing this issue.
